modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -89,3 +89,33 @@
 		cursor: pointer;
```

Comment:
@cgmonte You have some typos in the word `accessibility` in two places of the commit message.

Reply:
thanks! I will look into this.